docs/source/configuration.rst

```diff
@@ -88,6 +109,79 @@ in projects which utilise a lot of complicated templating.
 
 .. _templateconfig:
 
+Rule Configuration
+------------------
+
+Rules can be configured with the :code:`.sqlfluff` config files.
+
+Common rule configurations can be set in the :code:`[sqlfluff:rules]` section.
+
+For example:
+
+.. code-block:: cfg
+
+   [sqlfluff:rules]
+   tab_space_size = 4
+   max_line_length = 80
+   indent_unit = space
+   comma_style = trailing
+   allow_scalar = True
+   single_table_references = consistent
+   unquoted_identifiers_policy = all
+
+Rule specific configurations are set in rule specific subsections.
+
+For example, enforce that keywords are upper case by configuring the rule
+:class:`L010 <sqlfluff.core.rules.Rule_L010>`:
+
+.. code-block:: cfg
+
+    [sqlfluff:rules:L010]  # Keywords
+    capitalisation_policy = upper
+
+All possible options for rule sections are documented in :ref:`ruleref`.
+
+For an overview of the most common rule configurations that you may want to
+tweak, see `Default Configuration`_ (and use :ref:`ruleref` to find the
+available alternatives).
+
+Enabling and Disabling Rules
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+To disable individual rules, set :code:`exclude_rules` in the top level section
+of sqlfluff configuration. The value is a comma separated list of rule ids.
+
+For example, to disable the rules :class:`L010 <sqlfluff.core.rules.Rule_L022>`
+and :class:`L010 <sqlfluff.core.rules.Rule_L027>`:
+
+.. code-block:: cfg
+
+    [sqlfluff]
+    exclude_rules = L022, L027
+
+To enable individual rules, configure :code:`rules`, respectively.
+
+For example, to enable :class:`L010 <sqlfluff.core.rules.Rule_L027>`:
+
+.. code-block:: cfg
+
+    [sqlfluff]
+    rules = L027
+
+^ **TODO** does this make sense only if overriding an exclusion from a higher
+level configuration? Or what does this conf mean? How is it interpreted if
+:code:`exclude_rules = None` but :code:`exclude_rules != None`?
```

Comment:
Or do you mean like this?:

```
Top-level folder .sqlfluffini: exclude_rules = None
      Sub-folder .sqlfluffini: exclude_rules != None
```

In that case the subfolder will override the top level folder config and for the queries in that solder folder they will exclude some rules. This can be handy if you know a folder can't handle a certain rule, but the rest of the project can.

Note also it's overridden completely as far as I know.

So, with this for example:

```
Top level folder .sqlfluffini: exclude_rules = L001`
      Sub Folder .sqlfluffini: exclude_rules = L002`
```

Queries in the sub folder would only exclude `L002` and would start to lint for `L001`. It would NOT exclude both because the closer config overrides, but is not cumulative.

Reply:
I replaced this paragraph with:

> If both :code:`exclude_rules` and :code:`rules` have non-empty value, then the
excluded rules are removed from the rules list. This allows for example
enabling common rules on top level but excluding some on subdirectory level.

I would say this is the reasonable level of detail. Thoughts?

Reply:
That works for me!